src/js/webrequest.js

```diff
@@ -172,6 +172,23 @@ function onBeforeSendHeaders(details) {
     } else {
       return {};
     }
+
+  } else if (badger.isSpoofReferrerEnabled()) {
+    if (badger.isPrivacyBadgerEnabled(tabDomain) && badger.isPrivacyBadgerEnabled(requestDomain)) {
```

Comment:
We shouldn't check `requestDomain` here, right? Privacy Badger's site whitelist is for first-party (site) domains.